bravado/mapping/docstring.py

```diff
@@ -58,3 +58,65 @@ def create_model_docstring(props):
             docstring += ": " + props[prop]['description']
         docstring += '\n\t'
     return docstring
+
+
+def create_operation_docstring(json_):
+    """Builds Operation docstring from the json dict
+
+    :param json_: data to create docstring from
+    :type json_: dict
```

Comment:
json dict is pretty generic (and not really relevant to this function).

I had been changing these to "spec" or something like that.  Maybe "operation_definition" ?